### PR TITLE
Fix #980: In HDPI device shadow looks odd for edit profile image thumbnail.

### LINKED ISSUE #980: In HDPI device shadow looks odd for edit profile image thumbnail.
In HDPI device shadow looks odd for edit profile image thumbnail in profile progress activity.
Tested on Nexus S.
Screenshot 2020-04-14 at 11 55 27 AM
https://github.com/oppia/oppia-android/pull/998#discussion_r411155467

### PR DESCRIPTION
**Pull request by anandwana001**
## Explanation

Fixes #980 

Screenshot_1621492060
Screenshot_1621492049
Screenshot_1621491978
Screenshot_1621491971

### How to review
Check if the shadow is looking good in the following configurations
1. HDPI device - port and land
2. XHDPI device - port and land
3. XXXhdpi device - port and land

## Checklist

- [x] The PR title starts with "Fix #bugnum: ", followed by a short, clear summary of the changes. (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [x] The PR explanation includes the words "Fixes #bugnum: ..." (or "Fixes part of #bugnum" if the PR only partially fixes an issue).
- [x] The PR follows the [style guide](https://github.com/oppia/oppia-android/wiki/Coding-style-guide).
- [x] The PR does not contain any unnecessary auto-generated code from Android Studio.
- [x] The PR is made from a branch that's **not** called "develop".
- [x] The PR is made from a branch that is up-to-date with "develop".
- [x] The PR's branch is based on "develop" and not on any other branch.
- [x] The PR is **assigned** to an appropriate reviewer in both the **Assignees** and the **Reviewers** sections.

### CODE REVIEW

**Review by Sparsh1212 — COMMENTED** [review]
LGTM for xxxhdpi.
But I am still encountering the odd shadow on the edit image thumbnail for some devices. Can you please cross-check on your local?
1. Nexus S (HDPI)

image

2. Nexus 4 (xhdpi)

image

File: app/src/main/res/layout-sw600dp-port/profile_progress_header.xml [hunk 1]
```diff
@@ -46,13 +47,13 @@
       android:layout_width="0dp"
       android:layout_height="wrap_content"
       android:layout_marginStart="44dp"
-      android:layout_marginEnd="42dp"
       android:layout_marginTop="12dp"
+      android:layout_marginEnd="42dp"
       android:fontFamily="sans-serif-medium"
       android:text="@{viewModel.profile.name}"
+      android:textAlignment="viewStart"
```
Comment (rt4914) [inline app/src/main/res/layout-sw600dp-port/profile_progress_header.xml:54]: Remove this. Though it was added but for all xml files we need to remove `textAlignment`, `textDirection` and `textLocale`. Though we are not doing it right away but whenever we see this in code we should remove it.

File: app/src/main/res/layout-sw600dp-land/profile_progress_header.xml [hunk 2]
```diff
@@ -45,14 +46,14 @@
       android:id="@+id/profile_name_text_view"
       android:layout_width="0dp"
       android:layout_height="wrap_content"
+      android:layout_marginStart="44dp"
       android:layout_marginTop="12dp"
+      android:layout_marginEnd="202dp"
       android:fontFamily="sans-serif-medium"
       android:text="@{viewModel.profile.name}"
+      android:textAlignment="viewStart"
```
Comment (rt4914) [inline app/src/main/res/layout-sw600dp-land/profile_progress_header.xml:54]: Ditto

**Review by rt4914 — APPROVED** [review]
LGTM, thanks.

**Review by rt4914 — CHANGES_REQUESTED** [review]
Blocking on @Sparsh1212 suggestions.

**Comment by anandwana001** [comment]
closing this PR as not working on this currently

**Pull request closed without merging**